.github/workflows/rubocop.yml

```diff
@@ -84,15 +98,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, windows]
-        ruby: ['2.7', '3.0', '3.1', '3.2', '3.3', 'head']
-        include:
-          - os: ubuntu
-            ruby: jruby-9.4
-          - os: windows
-            ruby: mingw
-        exclude:
-          - os: windows
-            ruby: head
+        ruby: ['2.7', '3.3'] # Oldest and newest supported stable version
```

Comment:
Maybe you can also just use `ruby` here? Since it'll be bumped straight away anyways when a new one releases

Reply:
Good idea, I’ll do that.